src/platform/web/ui/session/room/MessageComposer.js

```diff
@@ -105,4 +110,13 @@ export class MessageComposer extends TemplateView {
             this._attachmentPopup.showRelativeTo(evt.target, 12);
         }
     }
+
+    _adjustHeight() {
+        window.requestAnimationFrame(() => {
+            this._input.style.height = "auto";
```

Comment:
This results in a slight change in height that causes the timeline layout to reflow but it does not trigger the ResizeObserver watching the timeline and as a result there is no call to `restoreScrollPosition`.
end result: the timeline is scrolled up a little.
I wonder if this has something to do with the browser render pipeline :thinking: